site/docs/pages/index.mdx

```diff
@@ -114,12 +197,19 @@ bun add @coinbase/onchainkit
       </App>
       </div>
     </div>
+
   </aside>
-  <aside className="md:px-2 pt-10">
-    <p className="text-lg md:text-xl py-4">
-      Create or connect your wallet with [Connect Wallet](/wallet/wallet), powered by [Smart Wallet](https://www.smartwallet.dev/why).
+  <main className="flex flex-col gap-4 md:flex-row pt-10">
+    <h3 className="basis-1/2 py-[15px] text-4xl md:text-4xl text-gray-50">Wallet</h3>
+  </main>
+  <aside>
+    <p className="text-base md:text-base pb-[24px]">
+      Create or connect your wallet with [Connect Wallet](/wallet/wallet), powered by [Smart Wallet](https://www.smartwallet.dev/why) and Paymaster.
```

Comment:
It does not feature Paymaster yet :) 

So we want to make sure we don't mention things we don't do yet.